includes/multi-currency/class-currency.php

```diff
@@ -19,35 +19,35 @@ class Currency {
 	 *
```

Comment:
In the `__construct` for this class, perhaps the default value for rate should be a float `1.00` rather than the string `''`?